### ISSUE TITLE
2.0 install fails: /usr/local/lib/vst/spectacle-analyzer-vst.so: No such file or directory

### ISSUE THREAD

**Issue by yurivict** [jpcima/spectacle#19]
```
Writing presets.ttl... done!
gmake install -C plugins/spectacle;
gmake[3]: Entering directory '/disk-samsung/freebsd-ports/audio/spectacle-lv2/work/spectacle-2.0/plugins/spectacle'
install: /disk-samsung/freebsd-ports/audio/spectacle-lv2/work/stage/usr/local/lib/vst/spectacle-analyzer-vst.so: No such file or directory
gmake[3]: *** [Makefile:189: install] Error 71
gmake[3]: Leaving directory '/disk-samsung/freebsd-ports/audio/spectacle-lv2/work/spectacle-2.0/plugins/spectacle'
gmake[2]: *** [Makefile:55: install] Error 2
gmake[2]: Leaving directory '/disk-samsung/freebsd-ports/audio/spectacle-lv2/work/spectacle-2.0'
*** Error code 2
```

OS: FreeBSD 12.2

**Comment by yurivict** [comment]
Built files are under ```bin```:
```
$ find . -name "*so"
./work/spectacle-2.0/bin/spectacle-analyzer-vst.so
./work/spectacle-2.0/bin/spectacle-analyzer.lv2/spectacle-analyzer.so
```

**Comment by jpcima** [comment]
This should be fixed by 27b1525

**Comment by yurivict** [comment]
Thanks.
